starboard/BUILD.gn

```diff
@@ -29,10 +29,12 @@ group("gn_all") {
     "//starboard/examples/hello_world:starboard_hello_world_example",
     "//starboard/examples/window:starboard_window_example",
     "//starboard/extension:extension_test",
-    "//starboard/loader_app:app_key_files_test",
     "//starboard/nplb",
     "//starboard/nplb/nplb_evergreen_compat_tests",
   ]
+  if (sb_is_evergreen_compatible) {
```

Comment:
maybe you need a "current_toolchain == starboard_toolchain" otherwise you might trigger this for modular builds building with cobalt_toolchain in a weird case.
https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/COBALT:starboard/linux/x64x11/shared/platform_configuration/configuration.gni;l=29?q=repo:lbshell-internal%2Fcobalt_src%20AND%20branch:COBALT%20sb_is_evergreen_compatible%20lang:gn

Reply:
Done that part